build.gradle

```diff
@@ -1,7 +1,7 @@
 apply plugin: "java"
 apply plugin: "maven"
 
-version = "2.5.0"
+version = "2.6.0"
```

Comment:
Given that there's no feature changes I don't think this warrants a version change. Usually version changes are only made when I make a release

Reply:
Version change was made with the intention I'd be updating the plugin to 1.15 since the readme was wrong.

Reply:
In that case, you should probably revert it